### PR TITLE
Add NVM prompt

### LINKED ISSUE #97: Add NVM support
First of all, thanks for the amazing work on this theme!

Secondly, I'd like to request a Node Version Manager integration. I've noticed that there's only one theme that has this currently, and it'd be pretty helpful. 
I can add it myself over the next weekend if nobody wants to take it. I can add it like the ruby/python managers and make it optional with an env var.

### PR DESCRIPTION
**Pull request by pioneerskies**
As it says. Should fix #97 

Sincerly I'm not 100% sure about colors I've used for all the themes: I chose the green and applied it down to all the supported color schemes.

Since this is a PR on a project I do not maintain, I decided to have this prompt disabled by default.

If NVM version is deactivated (`nvm deactivate`) the nvm prompt won't show. I'm open to other/smarter methods to understand if we're not on an nvm's node version.

Documentation should be ok also.

Note: this is a specific PR done for NVM, hence it's not meant to supersede the (stalled?) #128

### CODE REVIEW

**Comment by pioneerskies** [comment]
Sry for the push-force, but I've rebased after a fixup in order to retain clean history: I left a debug `echo` in the code. My bad.

**Comment by pioneerskies** [comment]
I've opted to not show the prompt also if `system` version is used

File: functions/__bobthefish_glyphs.fish [hunk 1]
```diff
@@ -62,6 +62,7 @@ function __bobthefish_glyphs -S -d 'Define glyphs used by bobthefish'
     set virtualenv_glyph \uE73C ' '
     set ruby_glyph       \uE791 ' '
     set go_glyph         \uE626 ' '
+    set node_glyph       \uF898 ' '
```
Comment (bobthecow) [inline functions/__bobthefish_glyphs.fish]: let's use `\uE718` instead? they look like they're the same, but `\uE718` has been in the fonts for a lot longer, so there's a better chance that existing users will have it in their patched fonts.

Comment (pioneerskies) [reply functions/__bobthefish_glyphs.fish]: Sure. Was not aware of the fact; thank you.

**Comment by bobthecow** [comment]
Thank you!

**Pull request merged**